[2024-03-24 08:41:09,760] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:40:00+00:00 [queued]>
[2024-03-24 08:41:09,999] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:40:00+00:00 [queued]>
[2024-03-24 08:41:10,001] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-24 08:41:10,001] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-03-24 08:41:10,002] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-24 08:41:10,234] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): transform_last_20> on 2024-03-24T08:40:00+00:00
[2024-03-24 08:41:10,344] {standard_task_runner.py:52} INFO - Started process 16955 to run task
[2024-03-24 08:41:10,351] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'transform_last_20', '2024-03-24T08:40:00+00:00', '--job-id', '776', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/file.py', '--cfg-path', '/tmp/tmp7u7zf_88', '--error-file', '/tmp/tmpc7yt3d_c']
[2024-03-24 08:41:10,352] {standard_task_runner.py:77} INFO - Job 776: Subtask transform_last_20
[2024-03-24 08:41:10,492] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.transform_last_20 2024-03-24T08:40:00+00:00 [running]> on host ce3400b6b8c4
[2024-03-24 08:41:10,770] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=transform_last_20
AIRFLOW_CTX_EXECUTION_DATE=2024-03-24T08:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-24T08:40:00+00:00
[2024-03-24 08:41:12,699] {python.py:151} INFO - Done. Returned value was: None
[2024-03-24 08:41:12,757] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=transform_last_20, execution_date=20240324T084000, start_date=20240324T084109, end_date=20240324T084112
[2024-03-24 08:41:12,871] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-24 08:41:12,934] {local_task_job.py:151} INFO - Task exited with return code 0
